docs/execute-methods.md

```diff
@@ -820,6 +820,18 @@ The response looks like `{"value":{"left":0,"top":96,"width":828,"height":1696}}
 
 `width` and `height` are the screen's width and height.
 
+### mobile: viewportScreenshot
+
+Takes a screenshot of the device viewport (see [`mobile: viewportRect`](#mobile-viewportrect))
+
+!!! warning "Unstable"
+
+    This method is unstable. We recommend using `mobile: viewportRect` instead
```

Comment:
viewportRect is also not good. And I would rather say "unreliable" instead of "unstable". It does not fail, just returns wrong/unexpected results under some circumstances.

Reply:
Maybe `getScreenshot` would be a better suggestion?